src/main/groovy/org/shipkit/internal/gradle/plugin/PluginDiscoveryPlugin.java

```diff
@@ -37,7 +35,7 @@ public void apply(final Project project) {
             public void execute(Plugin plugin) {
                 PluginBundleExtension extension = project.getExtensions().findByType(PluginBundleExtension.class);
 
-                Set<File> pluginPropertyFiles = discoverGradlePluginPropertyFiles(project);
+                Set<File> pluginPropertyFiles = PluginUtil.discoverGradlePluginPropertyFiles(project);
```

Comment:
At some point in the future we might look into how to push this bit into execution phase. For now, let's move on and keep plugin discovery plugin detect property files during the configuration phase.